Pipfile

```diff
@@ -7,7 +7,7 @@ name = "pypi"
 pylint = "*"
 
 [packages]
-cantools = "~=33.1.1"
+cantools = {git = "git://github.com/timbitss/cantools.git"}
```

Comment:
if we decide to go with the forking approach, i think its a better idea to fork into UBCFormulaElectric org and update the following line to `{git = "git://github.com/UBCFormulaElectric/cantools.git"}`